### PR TITLE
[resolver] Select a non-main branch option

### PR DESCRIPTION
**Pull request by ryx2**
**End-user friendly description of the problem this fixes or functionality that this introduces**

- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**

I usually do my development on a feature branch, but the current resolver only has the ability to develop on main

---
**Link of any specific issues this addresses**

### CODE REVIEW

**Comment by ryx2** [comment]
@neubig passes tests :)

**Review by neubig — COMMENTED** [review]
Hey thanks! I'm a bit confused by separating `source_branch` and `target_branch` though. Is there any situation under which you'd want to get the code from one branch and then send the PR back to a different branch? I'd be inclined to just have a single branch specified here for simplicity.

**Comment by ryx2** [comment]
@neubig hey, could i get this lgtm'd

**Comment by neubig** [comment]
Ahh, sorry, when you're ready to have it re-reviewed it helps if you click the "cycle" icon on the top right of the screen, which will put it back on my reviewing queue. Just did that now and I'll take a look.

File: .github/workflows/openhands-resolver.yml [hunk 1]
```diff
@@ -135,6 +140,9 @@ jobs:
           echo "MAX_ITERATIONS=${{ inputs.max_iterations || 50 }}" >> $GITHUB_ENV
           echo "SANDBOX_ENV_GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}" >> $GITHUB_ENV
 
+          # Set branch variables
+          echo "SOURCE_BRANCH=${{ inputs.target_branch }}" >> $GITHUB_ENV
```
Comment (neubig) [inline .github/workflows/openhands-resolver.yml:144]:
```suggestion
          echo "TARGET_BRANCH=${{ inputs.target_branch }}" >> $GITHUB_ENV
```

**Review by neubig — APPROVED** [review]
Fixed a typo, but other than that LGTM!

**Pull request merged**